<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Properties</class>
 <widget class="QWidget" name="Properties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QComboBox" name="FillPattern">
     <item>
      <property name="text">
       <string>Regular</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pattern</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDoubleSpinBox" name="BorderWidth"/>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="FillColor">
     <property name="text">
      <string>CHANGE COLOR</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="LabelBP">
     <property name="text">
      <string>Border Pattern</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="LabelBW">
     <property name="text">
      <string>Border Width</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="LabelBC">
     <property name="text">
      <string>Border Color</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="LabelFP">
     <property name="text">
      <string>Fill Pattern</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="BorderColor">
     <property name="text">
      <string>CHANGE COLOR</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="LabelFC">
     <property name="text">
      <string>Fill Color</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="BorderPattern">
     <item>
      <property name="text">
       <string>Regular</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dotted line</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
